modules/nw-operator-cr.adoc

```diff
@@ -449,41 +371,6 @@ defaultNetwork:
         mode: Full
 ----
 
-[discrete]
-[id="nw-operator-cr-kubeproxyconfig_{context}"]
-=== kubeProxyConfig object configuration (OpenShiftSDN container network interface only)
```

Comment:
(Even if you keep the minimal mention of `kubeProxyConfig` above, it may not be worth keeping the larger documentation of it here. Third-party plugins can document it themselves if they expect the admin to need to configure anything.)

Reply:
This is all coming out. Or are you saying there is more here you'd take out? @danwinship

Reply:
no, I was just referring to my [previous comment](https://github.com/openshift/openshift-docs/pull/79740#discussion_r1706983517), saying even if you keep the reference to `kubeProxyConfiguration` there, it still makes sense to delete it here.